shared/nav.ios.js

```diff
@@ -123,39 +123,16 @@ class Nav extends Component {
     )
   }
 
-  shouldComponentUpdate (nextProps, nextState) {
-    const activeTab = this.props.tabbedRouter.get('activeTab')
-    const nextActiveTab = nextProps.tabbedRouter.get('activeTab')
-    if (activeTab !== nextActiveTab) {
-      return true
-    }
-
-    if (this.props.config.navState !== nextProps.config.navState) {
-      return true
-    }
+  _activeTab () {
+    return this.props.tabbedRouter.get('activeTab')
+  }
 
-    return false
+  shouldComponentUpdate (nextProps, nextState) {
```

Comment:
also made this more like desktop's nav, not sending the entire store to this component